carla/recourse_methods/catalog/clue/library/clue_ml/src/utils.py

```diff
@@ -0,0 +1,320 @@
+from __future__ import division, print_function
+
+import os
+import sys
+
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+
+try:
+    import cPickle as pickle
+except:
+    import pickle
+
+
+def mkdir(paths):
```

Comment:
i'd rather not have library functionality that can read/write but control it ourselves